### ISSUE TITLE
IANA Section ADD the <dot> in the YANG model paragraph

### ISSUE THREAD

**Issue by stoffi92** [stoffi92/rfc5575bis#225]
The above textual changes clarify the definition of the SAFIs rather
   than change its underlying meaning.  Therefore, based on
   "The YANG 1.1 Data Modeling Language" [RFC7950], the above text
   implies that the following YANG descriptions from
   "Common YANG Data Types for the Routing Area" [RFC8294] need to have
   their descriptions at https://www.iana.org/assignments/iana-routing-
   types [2] changed to:

  ```
 <CODE BEGINS>
      enum ipv4-flow-spec-safi {
             value 133;
             description
               "Dissemination of Flow Specification rules SAFI.";
           }
      enum vpnv4-flow-spec-safi {
             value 134;
             description
               "L3VPN Dissemination of Flow Specification rules SAFI";
           }
   <CODE ENDS>

```
Just noticed there is one “.” <DOT> missing  in the description of value 134. I think it needs to be removed. -> will do in next revision.

**Comment by stoffi92** [comment]
@suehares 

I think that the suggested changes to the YANG Data types is not correct:

```
   enum ipv4-flow-spec-safi {
          value 133;
          description
            "Dissemination of Flow Specification rules SAFI.";
        }
   enum vpnv4-flow-spec-safi {
          value 134;
          description
            "L3VPN Dissemination of Flow Specification rules SAFI.";
        }

```

The SAFI is not limited to ipv4 anymore. How can we remove the "v4" also from the enum "names" - I am not an expert there but the name is not correct. I think it should be:

enum flow-spec-safi

and

enum l3vpn-flow-spec-safi

instead. Can you please suggest what needs to be changed in the text that I sent you for IANA review.

Christoph

**Comment by raszuk** [comment]
The reason for v4 there was to set clearly the next hop format against
which the validation is performed. If you remove that how do you know the
next hop format ?

**Comment by stoffi92** [comment]
There is nothing that we changed (recently).  It is only a required addtion to the YANG part that needed to be added because it was brought up by the last review. This case is about the YANG-code in IANA considerations.

The validation is based on the following:

 o  IPv4 Flow Specification identifier (AFI=1, SAFI=133) along with
      specific semantic rules for IPv4 routes, and

   o  VPNv4 Flow Specification identifier (AFI=1, SAFI=134) value, which
      can be used to propagate traffic filtering information in a BGP/
      MPLS VPN environment.

And in Section 6 (validation procedure):


   The validation process described below validates Flow Specifications
   against unicast routes received over the same AFI but the associated
   unicast routing information SAFI:

      Flow Specification received over SAFI=133 will be validated
      against routes received over SAFI=1

      Flow Specification received over SAFI=134 will be validated
      against routes received over SAFI=128

FS SAFI=133, 134 are also used for IPv6 … so having ipv4 in the name is not really a good idea.

This seems consistent.

Cheers Christoph

--
Christoph Loibl
c@tix.at | CL8-RIPE | PGP-Key-ID: 0x4B2C0055 | http://www.nextlayer.at

**Comment by suehares** [comment]
Christoph: 
My understanding is that the enums were on differnet trees - v4 and v6.  This is why I did not flag this earlier.  I will re-look at everything in a few minutes.

**Comment by stoffi92** [comment]
closed with commit 891e02d573388e475a6d0ef45aa92e13df679ae0
